app/models/user.rb

```diff
@@ -24,4 +27,26 @@ class User < ApplicationRecord
 
   mount_uploader :avatar, AvatarUploader
   paginates_per 10
+
+  def kid?
+    role.name == 'kid'
+  end
+
+  def elf?
+    role.name == 'elf'
+  end
+
+  def santa?
+    role.name == 'santa'
+  end
+
+  def admin?
+    role.name == 'admin'
+  end
+
+  private
+
+  def assign_role
+    self.role = Role.find_by(name: 'kid') if role.nil?
+  end
```

Comment:
If the role will be an enum, you will not need this all. 
`assign_role` can be replaced by assigning a default value in the migration